specification/datasets/cost_and_usage/columns/allocatedresourcedetails.md

```diff
@@ -0,0 +1,120 @@
+* AllocatedResourceDetails MUST conform to [StringHandling](#stringhandling) requirements.
+* AllocatedResourceDetails MUST conform to [ObjectFormat](#objectformat) requirements.
+  * AllocatedResourceDetails MUST conform to ObjectFormat schema requirements below.
+* AllocatedResourceDetails nullability is defined as follows:
+  * AllocatedResourceDetails MUST be null when a charge is not related to a provider-calculated split cost allocation.
+  * AllocatedResourceDetails SHOULD NOT be null when a charge is related to a provider-calculated split cost allocation.
+
+## Object Schema
+
+Allocated Resource Details consists of a valid JSON object which contains an array of key-value objects describing the one or more factors that determined the split cost allocation. Each object consists of FOCUS-defined keys but can be extended to provide additional details about the allocation.
+
+If AllocatedResourceDetails is not null, the ObjectFormat for AllocatedResourceDetails adheres to the following requirements:
+* ObjectFormat for AllocatedResourceDetails MUST be a valid JSON object.
+* AllocatedResourceDetails MUST have a top-level key "Elements" which contains an array.
+* Each item in "Elements" MUST be an object.
+  * Objects inside "Elements" MUST conform to [KeyValueFormat](#key-valueformat) requirements.
+  * Objects inside "Elements" MUST contain key-value pairs (allocation properties).
+    * FOCUS-defined allocation properties adhere to the following additional requirements:
+      * Allocation property key MUST match the spelling and casing specified for the FOCUS-defined property.
+      * Allocation property value MUST be of the type specified for that property.
+      * Allocation property MUST adhere to additional normative requirements specific to that property.
+    * Allocation property keys MUST begin with the string "x_" unless it is a FOCUS-defined allocation property.
+* AllocatedResourceDetails root object MAY contain additional items, in addition to "Elements".
+
+### FOCUS-Defined Allocation Properties
+
+The following keys are used for allocation properties to facilitate querying data across allocations and across providers. Focus-defined keys will appear in the list below and Provider-defined keys will be prefixed with "x_" to make them easy to identify as well as prevent collisions.
+
+FOCUS-defined allocation properties adhere to the following requirements:
+
+* "AllocatedRatio" MUST be included inside each "Elements" object.
+  * Values for "AllocatedRatio" MUST be a decimal value compatible with [NumericFormat](#numericformat) representing the allocated charge's percentage of the origin charge.
+  * Values for "AllocatedRatio" across all allocated records related to a single origin record MUST sum up to 1 (100%).
+* "UsageUnit" MUST be included inside an "Elements" object if "UsageQuantity" allocation property is included in that "Elements" object, otherwise "UsageUnit" MAY be included in each "Elements" object.
```

Comment:
Why not `ConsumedUnit` for consistency? Ideally we should be consistent if they're the same. If they're not the same, then we should probably come up with a different name because this will lead to confusion since people think of "ConsumedUnit" as "UsageUnit" already.